__tests__/app.test.js

```diff
@@ -173,3 +175,45 @@ describe("GET/api/articles", () => {
       });
   });
 });
+
+describe("GET/api/articles/:article_id/comments", () => {
+  test("returns status: 200 and returns all comemnts on any given article ", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((res) => {
+        const { comments } = res.body;
+
+        comments.forEach((comment) => {
+          expect(comment).toHaveProperty("comment_id");
+          expect(comment).toHaveProperty("votes");
+          expect(comment).toHaveProperty("created_at");
+          expect(comment).toHaveProperty("author");
+          expect(comment).toHaveProperty("body");
+          expect(comment).toHaveProperty("article_id");
+        });
```

Comment:
remember to check comments isn't just a empty arr and giving a false positive here!